### PR TITLE
Add additional AnyDesk paths

### PR DESCRIPTION
**Pull request by DevJoost**
(DIS-1333)

This adds some more possible file locations for relevant AnyDesk trace files. Also contains the default locations for the AnyDesk client/service on Linux/MacOS.

### CODE REVIEW

File: dissect/target/plugins/apps/remoteaccess/anydesk.py [hunk 1]
```diff
@@ -11,30 +11,43 @@
 class AnydeskPlugin(RemoteAccessPlugin):
     """
     Anydesk plugin.
+    
+    Trace file location based on https://support.anydesk.com/knowledge/trace-files#trace-file-locations
```
Comment (Schamper) [inline dissect/target/plugins/apps/remoteaccess/anydesk.py:15]:
```suggestion
```
You already list the URL under references down bellow, I don't think it's necessary to list it here too.

**Review by Schamper — CHANGES_REQUESTED** [review]
Can you also fix the linting issues (run black)?

File: dissect/target/plugins/apps/remoteaccess/anydesk.py (hunk 1 above)
Comment (DevJoost) [reply dissect/target/plugins/apps/remoteaccess/anydesk.py]: Agreed. Both should be fixed.

File: dissect/target/plugins/apps/remoteaccess/anydesk.py [hunk 2]
```diff
@@ -15,26 +15,37 @@ class AnydeskPlugin(RemoteAccessPlugin):
 
     __namespace__ = "anydesk"
 
-    # Anydesk log when service (Windows)
-    GLOBS = [
-        "/sysvol/ProgramData/AnyDesk/*.trace",
+    # Anydesk logs when installed as a service
+    SERVICE_GLOBS = [  # Only if service is installed
+        "/sysvol/ProgramData/AnyDesk/*.trace",  # Standard client >= Windows 7
+        "/sysvol/ProgramData/AnyDesk/ad_*/*.trace",  # Custom client   >= Windows 7
```
Comment (Schamper) [inline dissect/target/plugins/apps/remoteaccess/anydesk.py:21]:
```suggestion
        "/sysvol/ProgramData/AnyDesk/ad_*/*.trace",  # Custom client >= Windows 7
```

File: dissect/target/plugins/apps/remoteaccess/anydesk.py [hunk 3]
```diff
@@ -15,26 +15,37 @@ class AnydeskPlugin(RemoteAccessPlugin):
 
     __namespace__ = "anydesk"
 
-    # Anydesk log when service (Windows)
-    GLOBS = [
-        "/sysvol/ProgramData/AnyDesk/*.trace",
+    # Anydesk logs when installed as a service
+    SERVICE_GLOBS = [  # Only if service is installed
```
Comment (Schamper) [inline dissect/target/plugins/apps/remoteaccess/anydesk.py:19]:
```suggestion
    SERVICE_GLOBS = [
```
Previous comment should already be sufficient.

**Review by Schamper — CHANGES_REQUESTED** [review]

**Review by Schamper — APPROVED** [review]

**Pull request merged**